MisterFusion/MisterFusion.swift

```diff
@@ -248,8 +248,10 @@ extension UIView {
     public var CenterX: MisterFusion { return createMisterFusion(.CenterX) }
     
     public var CenterY: MisterFusion { return createMisterFusion(.CenterY) }
-    
+
+    #if swift(>=3.0)
     public var Baseline: MisterFusion { return createMisterFusion(.Baseline) }
```

Comment:
Thank you for PR!

But this line is not correct.

Swift2.2 has NSLayoutAttribute.Baseline.
<img width="417" alt="2016-09-10 2 53 25" src="https://cloud.githubusercontent.com/assets/2082134/18397209/20694afc-7702-11e6-852d-418566b54072.png">

Swift2.3 and up don't have NSLayoutAttribute.Baseline.
<img width="418" alt="2016-09-10 2 54 42" src="https://cloud.githubusercontent.com/assets/2082134/18397306/8725f9b6-7702-11e6-987b-ad7d4d5b2ab3.png">

Please check [https://developer.apple.com/reference/uikit/nslayoutattribute](https://developer.apple.com/reference/uikit/nslayoutattribute).